src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,15 @@
 package core.basesyntax;
 
 public class MainApp {
-
+    public static void main(String[] args) {
+        Mashine[] mashines = {
```

Comment:
The class name `Mashine` is consistently misspelled. It should be `Machine`. Please correct this typo in the array declaration and throughout the code.